doc/manual/citing_aspect.bib

```diff
@@ -1816,3 +1816,55 @@ @article{https://doi.org/10.1029/2022JB025036
 note = {e2022JB025036 2022JB025036},
 year = {2023}
 }
+
+@article{article,
+author = {Şengül Uluocak, Ebru},
+year = {2023},
+month = {09},
+pages = {},
+title = {A Discussion on Geodynamic Modeling Methodology: Inferences from Numerical Models in the Anatolian Plate},
+journal = {Türkiye Jeoloji Bülteni / Geological Bulletin of Turkey}
+}
```

Comment:
I got this from the publisher:
```suggestion
 @article { tjb1318091,
	journal = {Türkiye Jeoloji Bülteni/Geological Bulletin of Turkey},
	year = {2023},
	pages = {1 - 17},
	doi = {10.25288/tjb.1318091},
	title = {A Discussion on Geodynamic Modeling Methodology: Inferences from Numerical Models in the Anatolian Plate},
	key = {cite},
	author = {Şengül Uluocak,  Ebru}
}
```
I'm just going to accept my own suggestion and merge with this.